content/SCALECLIReference/Storage/CLIDisk.md

```diff
@@ -11,7 +11,624 @@ tags:
+The command returns a table containing the identifier, name, subsystem, number, serial, lunid, size, description, transfermode, hddstandby, advpowermgmt, togglesmart, smartoptions, expiretime, passwd, critical, difference, informational, model, rotationrate, type, kmip_uid, zfs_guid, bus, exported_zpool, unsupported_md_devices, duplicate_serial, enclosure, partitions, add devname properties for the specified disk.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>storage disk get_instance id="<i>diskidentifier</i>"</code>
+
+Where *diskidentifier* is the identifier of the disk you want to view.
+
+{{< expand "Command Example" "v" >}}
+
+```
+[test1234] storage disk> get_instance id="{serial_lunid}N4G22WKK_5000cca24503ce58"
++----------------+-----------------------------------------+
+|     identifier | {serial_lunid}N4G22WKK_5000cca24503ce58 |
+|           name | sdl                                     |
+|      subsystem | scsi                                    |
+|         number | 2224                                    |
+|         serial | N4G22WKK                                |
+|          lunid | 5000cca24503ce58                        |
+|           size | 2000398934016                           |
+|    description |                                         |
+|   transfermode | Auto                                    |
+|     hddstandby | ALWAYS ON                               |
+|   advpowermgmt | DISABLED                                |
+|    togglesmart | true                                    |
+|   smartoptions |                                         |
+|     expiretime | <null>                                  |
+|       critical | <null>                                  |
+|     difference | <null>                                  |
+|  informational | <null>                                  |
+|          model | HUS726020AL4210                         |
+|   rotationrate | 7200                                    |
+|           type | HDD                                     |
+|       zfs_guid | 5249139309134546836                     |
+|            bus | SCSI                                    |
+|        devname | sdl                                     |
+|      enclosure | <dict>                                  |
+| supports_smart | <null>                                  |
+|           pool | <null>                                  |
++----------------+-----------------------------------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Get_Unused Command
+
+The `get_unused` command returns disks that are not in use by any zpool. It will also return disks that are in use by any zpool that is exported. You can add the optional `join_partitions` argument to list disk partitions as well.
+
+{{< expand "Listing Unused Disks" "v" >}}
+
+#### Description
+The `get_unused` command does not require entering properties or arguments, but it does contain the optional `join_partitions` argument.
```

Comment:
```suggestion
The `get_unused` command does not have required properties , but it does have one optional property  `join_partitions` .
```
Below this add something like:
`join_partitions` has xxx properties. 
Or you can just tell the reader to go see the next section on using this property (which I just saw). 
Then enter the instruction for how to enter the property argument.